### DISCUSSION TITLE
Invalid Discord Link in Outreachy contribution page

### DISCUSSION THREAD

**Discussion by Ruchi-here** [pydata/xarray#7589] [General] [answered]
I tried joining the discord link which was given here
which was given here

![image](https://user-images.githubusercontent.com/122676573/223203293-089632e8-774c-4bf1-9206-2887a15f6824.png)

but its saying the link is invalid, so can you please tell how should I contact the mentor

**Comment by keewis** [comment]
we're all pretty new to `discord` so it seems we forgot to configure the link to not expire. In any case, here's a permanent invite: https://discord.gg/wEKPCt4PDu

**Comment by TomNicholas** [comment, marked as answer]
Hi @Ruchi-here - sorry about that! An updated [discord link is here](https://discord.gg/wEKPCt4PDu), and I'll update it on the outreachy site now. You can also tag me (@TomNicholas) anywhere you post or comment on this repository!

(Also the "Follow this link" re-directing to the issues page of this repository is intentional - that is where we do most of our communication!)

**Comment by Ruchi-here** [comment]
@TomNicholas sure, I'll check it out. I can help you with discord, if you need some help.

**Reply by husainridwan** [reply]
Hi, I'm trying to join the discord server but it's telling me "unable to accept invite". I'm using the link you provided above

**Reply by husainridwan** [reply]
@TomNicholas

**Reply by keewis** [reply]
that feels weird, since quite a few people have been able to join the server since we posted the permanent invite.

I'm really not certain, but looking at [some official documentation](https://support.discord.com/hc/en-us/articles/360001556852-Invalid-Invite-Links) I *think* that makes it an issue with your account (like, you joined about 100 servers already, or there's a login issue). The latter we can exclude if you're using the desktop app instead of the browser one.

**Reply by husainridwan** [reply]
Thanks. I’ve been able to join the discord now
